web/src/components/SecureAdminConsole.jsx

```diff
@@ -216,12 +225,12 @@ class SecureAdminConsole extends React.Component {
         token: token.replace(/"/g, ""),
         sessionRoles: Utilities.getCookie("session_roles"),
       };
-      const loggedIn = await this.completeLogin(loginData);
-      if (loggedIn) {
-        Utilities.removeCookie("token");
-        Utilities.removeCookie("session_roles");
-      }
-      return;
+      const loggedIn = this.completeLogin(loginData).then(() => {
```

Comment:
`componentWillMount` had an async function in it. I didn't do that for `componentDidMount` - I replaced the `await` with `.then`